x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/syncjob/ConnectorSyncJobTests.java

```diff
@@ -235,7 +235,7 @@ public void testFromXContent_WithAllNullableFieldsSetToNull_DoesNotThrow() throw
                 "metadata": {},
                 "started_at": null,
                 "status": "canceling",
-                "total_document_count": 0,
+                "total_document_count": null,
```

Comment:
Should we've two tests, one for `null` and for `0` just in case?

Reply:
Other tests are set to `0` , I only changed this test `testFromXContent_WithAllNullableFieldsSetToNull_DoesNotThrow` as it expects all nullable fields to be null